src/views/Dashboard.vue

```diff
@@ -0,0 +1,155 @@
+            create Portfolios to consolidate details about your project into a
+            single location, including:
+          </p>
+          <v-row class="pl-5">
+            <v-col class="stepper-img" cols="1">
+              <v-img
+                :src="require('../assets/dashboard/Step1.svg')"
+                contain
+                height="30"
+                min-width="25"
+              />
+            </v-col>
+            <v-col cols="5">
+              <p class="stepper-text">Task Orders</p>
+            </v-col>
+          </v-row>
+          <v-row class="pl-5">
+            <v-col class="stepper-img" cols="1">
+              <v-img
+                :src="require('../assets/dashboard/Step2.svg')"
+                contain
+                height="30"
+                min-width="25"
+              />
+            </v-col>
+            <v-col cols="5">
+              <p class="stepper-text">Application and Environments</p>
+            </v-col>
+          </v-row>
+          <v-row class="pl-5">
+            <v-col class="stepper-img" cols="1">
+              <v-img
+                :src="require('../assets/dashboard/Step3.svg')"
+                contain
+                height="30"
+                min-width="25"
+              />
+            </v-col>
+            <v-col cols="5">
+              <p class="stepper-text">Team Members</p>
+            </v-col>
+          </v-row>
+          <v-spacer></v-spacer>
+          <p class="dashboard-content">
+            Once your Portfolio is set up, you can submit this information to
+            selected cloud service providers, provisioning your workspaces and
+            teams, as well as establishing a funding stream between your
+            organization and the cloud provider(s).
+          </p>
+          <v-spacer></v-spacer>
+          <p class="dashboard-content">
+            Team members that develop your applications will then have access to
+            their workspaces within the CSP console, and you will be able to
+            track their cloud spend and contract period of performance directly
+            within ATAT.
+          </p>
+          <v-col cols="6">
+            <hr />
+          </v-col>
+          <h2 class="">Getting Started</h2>
+          <p class="dashboard-content">
+            First, you will need to verify your contact information that was
+            imported from your Active Directory user profile. Our administrators
+            may use this information to contact you, in the event that there is
+            an issue with your cloud resources or funding.
+          </p>
+          <v-container class="profile-card">
+            <v-row>
+              <v-col>
+                <h3 class="profile-card-name">Maria Missionowner</h3>
+              </v-col>
+            </v-row>
+            <v-row>
+              <v-col>
+                <p class="profile-card-text">maria.missionowner-civ@mail.mil</p>
```

Comment:
add identifier class or id for e2e
id="profile-card-text__email" 


repeat on similar items